test/Engine/FFITarget/DisposeTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace FFITarget
+{
+    public class DisposeTestClassA : IDisposable
+    {
+        static public int count { get; set; }
+
+        public DisposeTestClassA()
+        {
+            count = count + 1;
+        }
+
+        public void Dispose()
+        {
+            count = count - 1;
+        }
+    }
+
+    public class DisposeTestClassB : IDisposable
+    {
+        static public int count { get; set; }
+
+        public DisposeTestClassB()
+        {
+            count = count + 1;
+        }
+
+        public void Dispose()
+        {
+            count = count - 1;
+        }
+    }
+
+    public class DisposeTestClassC : IDisposable
+    {
+        static public int count { get; set; }
+
+        public DisposeTestClassC()
+        {
+            count = count + 1;
+        }
+
+        public void Dispose()
+        {
+            count = count - 1;
+        }
+    }
+
+    public class DisposeTestClassD : IDisposable
+    {
+        static public int s_dispose { get; set; }
```

Comment:
It was the same variable when it was still a DS defined class. I was trying to minimize the changes in the verification checks